crates/matrix-sdk-crypto/src/lib.rs

```diff
@@ -24,7 +24,7 @@ mod gossiping;
 mod identities;
 mod machine;
 pub mod olm;
-mod requests;
+pub mod requests;
```

Comment:
This is required because `matrix-sdk-crypto-js` needs to access some types from this module.

Reply:
`KeysBackupRequest`, `KeysQueryRequest`, `RoomMessageRequest` and `ToDeviceRequest`. It's not required in `matrix-sdk-crypto-ffi` because the types are not explicitely written down within the code. Let me search quickly.

In https://github.com/matrix-org/matrix-rust-sdk/blob/24b71bf869d717d438e7ed43bb978aa9def1c2ba/crates/matrix-sdk-crypto-ffi/src/responses.rs#L126-L163 for example, with `KeysBackup(b)`, `b` is of type `matrix_sdk_crypto::requests::KeysBackupRequets` but the type is never explicitly written down. However, in my implementation, I've taken another approach with https://github.com/Hywan/matrix-rust-sdk/blob/bb9ce3fae42f9b012b71150d236320a991a67120/crates/matrix-sdk-crypto-js/src/requests.rs#L168-L217 That way, I can build `*Request` types without going through `OutgoingRequest` everytime, e.g. https://github.com/Hywan/matrix-rust-sdk/blob/bb9ce3fae42f9b012b71150d236320a991a67120/crates/matrix-sdk-crypto-js/src/machine.rs#L311-L314